### ISSUE TITLE
No API response 

### ISSUE THREAD

**Issue by hunhar96** [davwheat/rail-announcements#279]
seems to be issues daily now with the live feed not returning results and showing no services available

**Comment by davwheat** [comment]
Yes, been ratelimited by the live departures API. Unfortunately no solution other than just waiting.

**Comment by davwheat** [comment]
This should now be resolved.

**Comment by hunhar96** [comment]
sadly re-occurred today mate

**Comment by Lucak1011** [comment]
This has returned.

**Comment by davwheat** [comment]
There was an issue before 2025 where two projects were sharing the same departures API token, which was triggering this. Now it just seems to be genuine users of only this site hitting the limits instead.

**Comment by hunhar96** [comment]
forgive my ignorance here of tech -but this there anything we can do other than stop using it?   can we obtain our own api token for example?

**Comment by davwheat** [comment]
I'd love to say exactly that, but Rail Delivery Group (RDG) have stopped accepting new users of the service the website uses (LDBSVWS - Live Departure Board Staff Version Web Service) and only accept subscriptions through the Rail Data Marketplace (RDM), which none of my code is set up to handle as it outputs in a completely different format.

The formats are _kind of_ similar, so it's possible I can do _something_ here and convert one format to the other, for example.

If I did that, though, I might as well migrate all my stuff to RDM since it _supposedly_ doesn't have a ratelimit (but I doubt this). There's been multiple "horror stories" of people suddenly having their access to services through RDM completely blocked or the services themselves breaking, which is why I'm reluctant to move over unnecessarily...

**Comment by AidanW1lson** [comment]
Issue has unfortunately returned in the past week or so. I presume owing to the above comment that the website itself will cease to function?

**Comment by davwheat** [comment]
It can be resolved but requires work which I don't have time to do at the moment.

**Comment by hunhar96** [comment]
Seems to be much smoother at the moment mate!

**Comment by hunhar96** [comment]
very odd, after weeks of really good reliable outputs, the dreaded API response keeps hitting again daily now.  very odd

**Comment by davwheat** [comment]
> I'd love to say exactly that, but Rail Delivery Group (RDG) have stopped accepting new users of the service the website uses (LDBSVWS - Live Departure Board Staff Version Web Service) and only accept subscriptions through the Rail Data Marketplace (RDM), which none of my code is set up to handle as it outputs in a completely different format.
> 
> The formats are _kind of_ similar, so it's possible I can do _something_ here and convert one format to the other, for example.

I have now done this. The site seems to be performing fine now.
